applications/nrf5340_audio/src/modules/nrf5340_audio_dk.h

```diff
@@ -0,0 +1,12 @@
+/*
+ * Copyright (c) 2023 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+/**
+ * @brief	Initialize the hardware related modules that are common for all the audio samples.
```

Comment:
```suggestion
 * @brief	Initialize the hardware related modules on the nRF5340 Audio DK/PCA10121.
```
Soon, it may not be common for "all" audio samples.

Reply:
Not sure I get what you mean, the comment was changed from "all audio samples" to just be the ones relevant for the nrf5340_audio DK